SonicWall/sonicwall-fw/_meta/fields.yml

```diff
@@ -0,0 +1,130 @@
+sonicwall.fw.appcat:
+    name: sonicwall.fw.appcat
+    description: "Application Control: Display the application category when Application Control is enabled"
+    type: keyword
+    
+sonicwall.fw.cat:
+    name: sonicwall.fw.cat
+    description: "Message category (legacy only): Indicates the legacy category number."
+   type: number
+
+sonicwall.fw.catid:
+    name: sonicwall.fw.catid
+    description: "Indicates the category ID of the rule"
+    type: number
+
+sonicwall.fw.cn1:
+    name: sonicwall.fw.cn1
+    description: "Indicates the ICMP type"
+    type: number
+
+sonicwall.fw.cn2:
+    name: sonicwall.fw.cn2
+    description: "Indicates the ICMP code"
+    type: number
+
+sonicwall.fw.cnt:
+    name: sonicwall.fw.cnt
+    description: "Message count: Indicates the number of times event occurs"
+    type: number
+
+sonicwall.fw.cs2:
+    name: sonicwall.fw.cs2
+    description: "Source VPN policy name: Displays the source VPN policy name
+of event"
+    type: keyword
+
+sonicwall.fw.cs3:
+    name: sonicwall.fw.cs3
+    description: "Destination VPN policy name: Displays the destination VPN policy name of event"
+    type: keyword
+
+sonicwall.fw.cs3Label:
+    name: sonicwall.fw.cs3Label
+    description: "Source zone name: Displays source zone"
+    type: keyword
+
+sonicwall.fw.cs4:
+    name: sonicwall.fw.cs4
+    description: "Interface Statistics: Display interface statistics"
+    type: keyword
+
+sonicwall.fw.cs4Label:
+    name: sonicwall.fw.cs4Label
+    description: "Destination zone name"
+    type: keyword
+
+sonicwall.fw.cs5:
+    name: sonicwall.fw.cs5
+    description: "URL: Applicable only when Network Packet Capture System (NPCS Solera) is enabled, displays URL of an NPCS object"
+    type: keyword
+
+sonicwall.fw.cs5Label:
+    name: sonicwall.fw.cs5Label
+    description: "Pre-defined string indicating session type: Applies to Syslogs with an associated user session being tracked by the UTM. "
+    type: keyword
+
+sonicwall.fw.cs6:
```

Comment:
Those fields are CEF specific and might not contain the same information for each event. Please use `.get('<FIELD NAME>)` to collect the content of the field.

Reply:
SonicWall does not conform to the CEF norm, you can see in their [documentation](https://www.sonicwall.com/techdocs/pdf/sonicos-6-5-4-log-events-reference-guide.pdf) that their fields do not correspond to the labels.